doc/RecordingControl.xml

```diff
@@ -1765,6 +1797,17 @@ Change Request 2061, 2063, 2065, 2109</revremark>
             <para role="text">A value indicating that the export has reached the maximum count and a new request must be sent to query the export state of further segments.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term>faults</term>
+          <listitem>
+            <para role="param">env:Sender - ter:InvalidArgVal - ter:NoOperation</para>
+            <para role="text">The referenced operation does not exist.</para>
+            <para role="param">env:Sender - ter:InvalidArgVal - ter:InvalidStorage</para>
```

Comment:
I don't think this fault makes much sense, since the request does not specify any storage configuration.